gradle.properties

```
@@ -29,3 +29,5 @@ kotlin.mpp.enableCInteropCommonization=true
 # Workaround for Bintray treating .sha512 files as artifacts
 # https://github.com/gradle/gradle/issues/11412
 systemProp.org.gradle.internal.publish.checksums.insecure=true
+
+org.gradle.jvmargs=-Xmx2g -XX:MaxMetaspaceSize=2048m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
```

```suggestion
org.gradle.jvmargs=-Xmx2g -XX:MaxMetaspaceSize=2g -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
```
Just for consistency.

Also, why do we need to specify file encoding explicitly? And why did everything work somehow without it before?

---

We don't need to set the encoding, I just read gradle docs and left the config from the example as it was. We don't need a heap dump either.